lib/module/xkcd.go

```diff
@@ -0,0 +1,70 @@
+package module
+
+import (
+	"fmt"
+	"github.com/davidscholberg/irkbot/lib/configure"
+	"github.com/davidscholberg/irkbot/lib/message"
+	"github.com/nishanths/go-xkcd"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+const apiUrlFmtDefine = "https://relevantxkcd.appspot.com/process?%s"
+
+func Helpxkcd() []string {
+	s := "xkcd <search> - find a xkcd comic relevant to the search term"
+	return []string{s}
+}
+func get(apiURL string) (string, error) {
+	response, err := http.Get(apiURL)
+	if err != nil {
+		return "", err
+	}
+	defer response.Body.Close()
+	bodyBytes, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		return "", err
+	}
+	bodyString := string(bodyBytes)
+	return bodyString, err
```

Comment:
There's no need to return the err object here, since there was no error. It's clearer if you just do `return bodyString, nil`.